### DISCUSSION TITLE
Git commit without Signed-off-by

### DISCUSSION THREAD

**Discussion by nlamirault** [renovatebot/renovate#9908] [Request Help] [answered]
**What Renovate type, platform and version are you using?**

**Renovate type**

- [x] Hosted app
- [ ] Self hosted

**Renovate platform**

- [ ] Azure DevOps (dev.azure.com)
- [ ] Azure DevOps Server (self-hosted)
- [ ] Bitbucket Cloud (bitbucket.org)
- [ ] Bitbucket Server (self-hosted)
- [ ] Gitea (gitea.com, self-hosted)
- [x] GitHub (github.com, GitHub Enterprise)
- [ ] GitLab (gitlab.com, self-hosted)

**Renovate version (if self hosting)**

I'm self hosting Renovate with version: ...

**Describe the bug**

With same configuration some PR are git sign and others not.

Ex:

* https://github.com/nlamirault/terraform-google-vector/pull/10
* https://github.com/nlamirault/terraform-aws-vector/pull/11

Sign : https://github.com/nlamirault/terraform-google-vector/pull/10/commits/80e33267aea37c0f68712179eb9612c80b3290f8
Not sign : https://github.com/nlamirault/terraform-aws-vector/pull/11/commits/f0428e3cf16e2f68b1943e9d5308030580804a65

```
$ diff ./terraform-google-vector/.github/renovate.json5 ./terraform-aws-vector/.github/renovate.json5 
$ diff ./terraform-google-vector/.github/workflows/renovate.yml ./terraform-aws-vector/.github/workflows/renovate.yml 
```
Project have same configurations. 
Any idea why i've got this problem ?

Thanks.

**Relevant debug logs**

<details><summary>Click me to see logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [ ] I have provided a minimal reproduction repository
- [x] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

**Comment by rarkins** [comment, marked as answer]
It's because you configured git signoff in one repo but not in the other

**Reply by nlamirault** [reply]
@rarkins Configuration files renovate.json5 are identical in the 2 repositories. Both have : 
```json
"extends": [
    "config:base",
    ":gitSignOff"
  ],
```
Is there another

**Reply by rarkins** [reply]
You need to delete https://github.com/nlamirault/terraform-aws-vector/blob/master/renovate.json because it will take precedence over your other config file

**Reply by nlamirault** [reply]
Thanks. I will do that @rarkins . Sorry for noise.

**Reply by viceice** [reply]
Here's the config file precedence:

https://github.com/renovatebot/renovate/blob/7b078e0f1c872daa1922a52a5de3e79a618e4386/lib/config/app-strings.ts#L1-L10

Only first match is used.
